src/topology/algebra/mul_action2.lean

```diff
@@ -0,0 +1,210 @@
+/-
+Copyright (c) 2021 Alex Kontorovich, Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex Kontorovich, Heather Macbeth
+-/
+import topology.homeomorph
+import group_theory.group_action.basic
+/-!
+# Continuous monoid action
+
+In this file we define class `has_continuous_smul₂`. We say `has_continuous_smul₂ Γ T` if `Γ` acts
+on `T` and for each `γ`, the map `x ↦ γ • x` is continuous.
```

Comment:
I think it's probably worth dropping a few lines here about why this is different to `has_continuous_smul` so that there's no need to google that!